### PR TITLE
Fix issue #63: Improve tone and fix typo in 02_minimal_measures.md

### PR DESCRIPTION
**Pull request by SalmanDeveloperz**
This PR addresses issue #63 by making improvements to the text in the 02_minimal_measures.md file.

### CODE REVIEW

**Comment by SalmanDeveloperz** [comment]
Hi,
I’ve made some updates to the 02_minimal_measures.md file. Please feel free to let me know if anything else is needed or if you have any suggestions.
Looking forward to your feedback!
Thanks,

File: DISCOVER/02_minimal_measures.md [hunk 1]
```diff
@@ -3,11 +3,11 @@
 
 ***As a general rule, we encourage conference organizers to implement all the 🍎 measures and at least one of the ✅ suggestions.***
 
-Making your conference inclusive and encouraging diversity is a breeze!
+Making your conference inclusive and encouraging diversity is a achievable. The measures below are explained in more depth in the other pages of this Cookbook.
```
Comment (aterrel) [inline DISCOVER/02_minimal_measures.md:6]: "is a achievable" isn't a complete phrase.

**Review by aterrel — CHANGES_REQUESTED** [review]

File: DISCOVER/02_minimal_measures.md (hunk 1 above)
Comment (mikofski) [reply DISCOVER/02_minimal_measures.md:6]: I think it’s meant to be
> Making your conference inclusive and encouraging diversity is achievable.

I think it’s a typo and the “a” was just left over? Remove it and it makes more sense to me.

```suggestion
Making your conference inclusive and encouraging diversity is achievable. The measures below are explained in more depth in the other pages of this Cookbook.
```

Comment (mikofski) [reply DISCOVER/02_minimal_measures.md:6]: Also this repo is set up with a “preview” action that renders the Jupyter book in the PR, someone with “write” access has to approve the workflow to enable it.

Comment (SalmanDeveloperz) [reply DISCOVER/02_minimal_measures.md]: Hi, I've made the requested updates. Please let me know if further modifications are needed.
Thanks

**Review by aterrel — APPROVED** [review]

**Review by aterrel — APPROVED** [review]

**Comment by tkoyama010** [comment]
@all-contributors please add @SalmanDeveloperz for doc

**Comment by tkoyama010** [comment]
@all-contributors please add @aterrel for maintenance and review

**Pull request merged**